tests/aws-grow-cluster-200nodes.yaml

```diff
@@ -0,0 +1,61 @@
+test_duration: 1440
+cassandra_stress_threads: 500
+cassandra_stress_population_size: 10000000
+nemesis_interval: 1
+user_prefix: 'cluster-scale-test'
+failure_post_behavior: destroy
+space_node_threshold: 100
+cluster_initial_size: 50
+cluster_target_size: 200
+add_node_cnt: 1
+experimental: 'true'
+
+backends: !mux
+    aws: !mux
+        # What is the backend that the suite will use to get machines from.
+        cluster_backend: 'aws'
+        # From 0.19 on, iotune will require bigger disk, so let's use a big
+        # loader instance by default.
+        instance_type_loader: 'c4.4xlarge'
+        # Size of AWS monitor instance
+        instance_type_monitor: 'c4.2xlarge'
+        us_east_1:
+            region_name: 'us-east-1'
+            security_group_ids: 'sg-c5e1f7a0'
+            subnet_id: 'subnet-d934e980'
+            ami_id_db_scylla: 'ami-7e137d68'
+            ami_db_scylla_user: 'centos'
+            ami_id_loader: 'ami-7e137d68'
+            ami_loader_user: 'centos'
+            ami_id_db_cassandra: 'ami-3eff0028'
+            ami_db_cassandra_user: 'ubuntu'
+            ami_id_monitor: 'ami-3d101f2a'
+            ami_monitor_user: 'centos'
+    gce: !mux
```

Comment:
Does this test work also in gce?
Your yaml file called aws-... but you have a mux also for gce.

IIRC, it took very long time to run it in gce due to all installations we are doing per node.